exec/Main.hs

```diff
@@ -148,6 +152,24 @@ addEventsModuleNameIndex =
     , statement = "CREATE INDEX IF NOT EXISTS events_height_chainid_module ON events (height DESC, chainid, module);"
     }
 
+addFromAccountsIndex :: LogFunctionIO Text -> Connection -> IO ()
+addFromAccountsIndex =
+  addIndex
+    IndexCreationInfo
+    {
+      message = "Adding \"(from_acct)\" index on transfers table"
+    , statement = "CREATE INDEX IF NOT EXISTS ON transfers (from_acct);"
```

Comment:
There's a SQL syntax error here, `IF NOT EXISTS` need to be followed by the name of the index. The same goes for `addToAccountsIndex` as well.

Reply:
I've resolved this issue.